phpunit/class-rest-sidebars-controller-test.php

```diff
@@ -274,6 +289,24 @@ public function test_get_item() {
 		);
 	}
 
+	/**
+	 * @dataProvider users_without_permissions
+	 */
+	public function test_get_item_permission( $user_id ) {
+		wp_set_current_user( $user_id );
+		$this->setup_sidebar(
+			'sidebar-1',
+			array(
+				'name' => 'Test sidebar',
+			)
+		);
+
+		$request  = new WP_REST_Request( 'GET', '/__experimental/sidebars/sidebar-1' );
+		$response = rest_get_server()->dispatch( $request );
+		$this->assertEquals( 401, $response->get_status() );
+		$this->assertErrorResponse( 'widgets_cannot_access', $response, 401 );
```

Comment:
```suggestion
		$this->assertErrorResponse( 'widgets_cannot_access', $response, 403 );
```

A `401` is issued if the user is logged out, a `403` if they don't have permission. See `rest_authorization_code_required`.